apps/src/standaloneVideo/StandaloneVideo.tsx

```diff
@@ -0,0 +1,49 @@
+// StandaloneVideo
```

Comment:
minor nit: left a similar [comment](https://github.com/code-dot-org/code-dot-org/pull/52552#discussion_r1253446934) on Molly's PR, but should we standardize where we put top-level docs for react functional components? i have a slight preference for putting them after imports, on top of the component declaration, but could go either way.

Reply:
I also don't have strong feelings, but I think there is [one advantage](https://github.com/code-dot-org/code-dot-org/pull/52552#discussion_r1253590619) to having them at the top of the file.